Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -165,11 +172,11 @@ void HiresTexture::Prefetch()
 				// But bad luck, SOIL isn't, so TODO: remove SOIL usage here and use libpng directly
 				// Also TODO: remove s_textureCacheAquireMutex afterwards. It won't be needed as the main mutex will be locked rarely
 				//lk.unlock();
-				HiresTexture* t = Load(base_filename, 0, 0);
+				std::unique_ptr<HiresTexture> texture = Load(base_filename, 0, 0);
```

Comment:
This unique ptr will never call "delete", so I'm not sure if it's worth this effort.